vars/kola.groovy

```diff
@@ -74,21 +74,36 @@ def call(params = [:]) {
             args += " --exttest ${env.WORKSPACE}/${path}"
         }
 
-        // basic run
-        if (!params['skipBasicScenarios']) {
-            id = marker == "" ? "kola-basic" : "kola-basic-${marker}"
+        if (platformArgs != "" || extraArgs != "") {
```

Comment:
There's the corner-case of if `platformArgs` is set for QEMU.

Reply:
I still kind of feel like that falls into the category of `"you're doing something special so you get the single run"`.

Reply:
I guess I could try to make the comment more broad

Reply:
Ack, I think that's reasonable. I'm OK not changing the comment.